Assets/__Scripts/UI/LightshowController.cs

```diff
@@ -18,10 +18,7 @@ public void OnToggleLightshowMode(InputAction.CallbackContext context)
     {
         if (context.performed)
         {
-            if (cameraController.transform.parent != null)
-            {
-                cameraController.OnAttachtoNoteGrid(context);
-            }
+            cameraController.LockedOntoNoteGrid = false;
```

Comment:
Could we have it remember the previous state, and re-apply that when lightshow mode is off? It would be a QOL thing mainly for 360/90 mode

Reply:
We can, just as per the original description this is how it was before.